public/video-ui/src/reducers/publishedVideoReducer.js

```diff
@@ -0,0 +1,13 @@
+export default function video(state = null, action) {
```

Comment:
I think https://stackoverflow.com/questions/5101948/javascript-checking-for-null-vs-undefined-and-difference-between-and explains it well. `null` is a bit more explicit, it's an empty object.